share/github-backup-utils/ghe-backup-es-audit-log

```diff
@@ -19,7 +19,7 @@ ghe_remote_version_required "$host"
 # Make sure root backup dir exists if this is the first run
 mkdir -p "$GHE_SNAPSHOT_DIR/audit-log"
 
-indices=$(ghe-ssh "$host" 'curl -s "localhost:9201/_cat/indices/audit_log*"' | cut -d ' ' -f 3)
+indices=$(ghe-ssh "$host" 'curl -s "localhost:9201/_cat/indices/audit_log*?h=index"') || true
```

Comment:
I'd prefer better error handling here and warn if we can't backup audit logs, instead of ignoring the error.